include/nuttx/usb/usbdev_trace.h

```diff
@@ -419,7 +419,7 @@ typedef uint16_t usbtrace_idset_t;
 
 /* Print routine to use for usbdev_trprint() output */
 
-typedef CODE int (*trprintf_t)(const char *fmt, ...);
+typedef CODE int (*trprintf_t)(const char *fmt, ...) printflike(1, 2);
```

Comment:
Is `printflike` applicable to typedefs?

Reply:
The compiler doesn't report the error, so it should be applicable.